### ISSUE TITLE
[v2-onboarding] setup-skill Step 3: no backup verification step for irrecoverable AAISN private key

### ISSUE THREAD

**Issue by dwinter3** [OneZero1ai/8th-layer-agent#138]
## Observed
After backing up the AAISN private key to Secrets Manager, the skill does not instruct the operator to read the secret back and verify it matches the local file. For an irrecoverable key (loss = permanent namespace forfeiture), a round-trip verify should be mandatory before proceeding.

## Fix
Add after the backup command:
```bash
aws secretsmanager get-secret-value --secret-id "$SECRET_ARN" --query SecretBinary --output text --profile $AWS_PROFILE | base64 -d > /tmp/key-verify && diff /tmp/key-verify ~/.config/8l-directory/keys/$SLUG.key && rm /tmp/key-verify && echo 'Backup verified ✅'
```

Source: cq-fanboy v2 onboarding, 2026-05-08
